core/src/main/java/io/kestra/core/serializers/FileSerde.java

```diff
@@ -1,26 +1,41 @@
 package io.kestra.core.serializers;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
+import static io.kestra.core.utils.Rethrow.throwBiFunction;
+import static io.kestra.core.utils.Rethrow.throwConsumer;
+import static io.kestra.core.utils.Rethrow.throwFunction;
+
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.MappingIterator;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SequenceWriter;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.FluxSink;
+import reactor.core.publisher.Mono;
 
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
 import java.io.BufferedReader;
 import java.io.IOException;
+import java.io.InputStream;
 import java.io.OutputStream;
 import java.io.UncheckedIOException;
+import java.util.Optional;
 import java.util.function.Consumer;
 
-abstract public class FileSerde {
-    private static final ObjectMapper MAPPER = JacksonMapper.ofIon()
-        .setSerializationInclusion(JsonInclude.Include.ALWAYS);
+public final class FileSerde {
+
+    private static final ObjectMapper DEFAULT_OBJECT_MAPPER = JacksonMapper.ofIon();
+    private static final TypeReference<Object> DEFAULT_TYPE_REFERENCE = new TypeReference<>(){};
+
+    /** The size of the buffer used for reading and writing data from streams. */
+    private static final int BUFFER_SIZE = 65535;
```

Comment:
Java default buffer size is 8k, we can use a bigger buffer but to balance performance with memory usage maybe x4 is enough. I also find it more easy to write such size by multiple of 1024


```suggestion
    private static final int BUFFER_SIZE = 32 * 1024;
```